packages/runtime/src/controls/style.ts

```diff
@@ -385,5 +385,132 @@ if (import.meta.vitest) {
       // Assert
       expect(result).toMatchObject(expected)
     })
+
+    test('colors are replaced - v1', () => {
```

Comment:
There should be no need for this test as the Style control should be using `ColorData`, not control specific types which is versioned.

Basically, this file shouldn't change since we're only updating the Color control, not the Style control